app/ml/classification_utils.py

```diff
@@ -83,3 +88,18 @@ def classify_image(model_id, img_id):
 
     img.close()
     return output
+
+def process_input_image(input):
+    if isinstance(input,Image.Image):  #pass directly the image
+        return input
+    elif isinstance(input,str):  #pass image id
+        return fetch_image(input)
+    else:
+        raise TypeError("FILE TYPE UNKNOWN")
+
+def check_format(img):
+    format = img.format.lower()
+    if format == 'jpeg' or format == 'png':
+        return True
```

Comment:
Suggestion (you don't have to change it): it's not safe to check the file format by only looking at its extension. A safer method is to check the magic number, for instance with [python-magic](https://github.com/ahupp/python-magic).

Reply:
done